pkg/instance/instance_test.go

```diff
@@ -29,11 +29,12 @@ func TestFuzzerCmd(t *testing.T) {
 	flagTest := flags.Bool("test", false, "enable image testing mode") // used by syz-ci
 	flagExecutor := flags.String("executor", "./syz-executor", "path to executor binary")
 	flagSignal := flags.Bool("cover", false, "collect feedback signals (coverage)")
-	flagSandbox := flags.String("sandbox", "none", "sandbox for fuzzing (none/setuid/namespace)")
+	flagSandbox := flags.String("sandbox", "none", "sandbox for fuzzing (none/setuid/namespace/android)")
+	flagSandboxArg := flags.Int("sandbox_arg", 0, "an argument to customize executor behavior")
```

Comment:
This test ensures that we are still capable of executing old fuzzer binaries. You shouldn't have added new flags here.